api.md

```diff
@@ -238,41 +238,55 @@ Add a new data node to the database to be used for creating
 distributed hypertables.
 
 Newly created distributed hypertables will be able to make use of this
-data node for apportioning data across the distributed database. Note that existing
-distributed hypertables will not automatically use the newly added
-data node. For existing distributed hypertables to use added data
-nodes, use [`attach_data_node`](#attach_data_node).
-
-The function will attempt to bootstrap the data node by:
-1. Connecting to a specified postgres instance
-2. Creating the database that will serve as the new data node
-3. Loading the TimescaleDB extension on the new database
+data node for apportioning data across the distributed database. Note
+that existing distributed hypertables will not automatically use the
+newly added data node. For existing distributed hypertables to use
+added data nodes, use [`attach_data_node`](#attach_data_node).
+
+If the data node already exists, the command will abort with either an
+error or a notice depending on the value of `if_not_exists`.
+
+If `bootstrap` is true, the function will attempt to bootstrap the
+data node by:
+1. Connecting to the database given in `bootstrap_database`.
+2. Creating the database given in `database` that will serve as the
+   new data node.
+3. Loading the TimescaleDB extension in the new database.
+4. Set the distributed UUID of the database to the UUID of the current
```

Comment:
```suggestion
4. Setting the distributed UUID of the database to the UUID of the current
```

Reply:
FWIW, this seems a bit too implementation specific for user docs. I would just say something like Installing the necessary metadata to make it part of a distributed database.